fil-proofs-tooling/Cargo.toml

```diff
@@ -49,6 +49,9 @@ humansize = "1.1.0"
 blstrs = "0.6.0"
 time = "0.3.9"
 
+[build-dependencies]
+vergen = "3.1.0"
```

Comment:
I only had a quick look. Would it be possible to use the most current version of `vergen`?

Reply:
Sure, Update current last version vergen@V8.1.1